cpu.Dockerfile

```
@@ -20,12 +20,17 @@ ENV UV_COMPILE_BYTECODE=1
 # Copy from the cache instead of linking since it's a mounted volume
 ENV UV_LINK_MODE=copy
 
-# Copy the application files into the container
-COPY . /app
+# Install the transitive dependencies
+RUN --mount=type=cache,target=/root/.cache/uv \
+    --mount=type=bind,source=uv.lock,target=uv.lock \
+    --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
+    uv sync --no-dev --extra cpu --extra frontend --locked --no-install-project
+
+ADD . /app
```

Hadolint (2.12.0)

[error] Use COPY instead of ADD for files and folders

(DL3020)